cypress/integration/links.ts

```diff
@@ -0,0 +1,11 @@
+describe('On links example', () => {
```

Comment:
The `describe` is the noun and the `it` is the verb. So like `describe('Monkey', () => { it('can climb')})`
```suggestion
describe('Links example', () => {
```